.github/workflows/ci-actions.yml

```diff
@@ -467,9 +467,10 @@ jobs:
             test-modules: >
               kubernetes-client
           - category: Misc4
-            timeout: 10
+            timeout: 30
```

Comment:
Is the 20min increase warranted by adding a single integration test?

Reply:
Last run was close to 13 min. I will set this to 20.